apps/student/src/app/documents/[documentId]/page.tsx

```
@@ -0,0 +1,377 @@
+                            <line x1="12" y1="3" x2="12" y2="15" />
+                          </svg>
+                        </span>
+                      )}
+                    </div>
+                    
+                    {selectedFile && (
+                      <div className="flex justify-end">
+                        <button 
+                          onClick={() => setSelectedFile(null)}
+                          className="text-[13px] text-red-500 hover:underline"
+                        >
```

Biome (2.4.15)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)